tests/deployment/serverless/test_serverless_utils.py

```diff
@@ -0,0 +1,14 @@
+from mock import patch
+import pytest
+
+from bentoml.deployment.serverless.serverless_utils import (
+    check_nodejs_compatible_version,
+)
+from bentoml.exceptions import BentoMLMissingDependencyException
+
+
+def test_check_nodejs_compatible_version():
+    with pytest.raises(BentoMLMissingDependencyException) as error:
+        with patch('bentoml.utils.whichcraft.which', return_value=None):
```

Comment:
Oh I think I know why that 'import' change was made now.. ignore that previous comment

Reply:
Would be great to add a comment at that function tho